examples/html2text.rs

```diff
@@ -7,88 +7,84 @@ use log::trace;
 use std::io;
 use std::io::Write;
 
-#[cfg(unix)]
 use html2text::render::RichAnnotation;
-#[cfg(unix)]
 fn default_colour_map(
     annotations: &[RichAnnotation],
     s: &str,
     use_css_colours: bool,
     no_default_colours: bool,
 ) -> String {
-    use termion::color::*;
+    use yansi::{hyperlink::HyperlinkExt, Paint};
     use RichAnnotation::*;
     // Explicit CSS colours override any other colours
     let mut have_explicit_colour = no_default_colours;
-    let mut start = Vec::new();
-    let mut finish = Vec::new();
+    let mut styled = s.to_string();
     trace!("default_colour_map: str={s}, annotations={annotations:?}");
     for annotation in annotations.iter() {
-        match annotation {
-            Default => {}
-            Link(_) => {
-                start.push(format!("{}", termion::style::Underline));
-                finish.push(format!("{}", termion::style::Reset));
-            }
-            Image(_) => {
+        styled = match annotation {
+            Default => styled,
+            Link(url) => styled.link(url).blue().underline().to_string(),
```

Comment:
I don't think the hyperlinks should be on by default, but fine with an opt in option (`--hyperlinks` or similar).